modules/wsl-distro.nix

```
@@ -42,6 +42,7 @@ in
             type = str;
             description = "The name the file should be created as in /bin";
             default = baseNameOf config.src;
+            defaultText = literalExpression "publicKey";
```

```suggestion
            defaultText = literalExpression "baseNameOf src";
```
Since src is dynamic we cannot hardcode this
```
[ { copy = false; name = "wslpath"; src = "/init"; } { copy = false; name = "sh"; src = "/nix/store/dw9nvc7s6x04p93gdmzkl7m0i03m8bhv-sh/bin/sh"; } { copy = false; name = "mount"; src = "/nix/store/0clijsbl2jw932bpskr5w21m01ajdqw8-util-linux-2.39.2-bin/bin/mount"; } { copy = true; name = "nixos-wsl-recovery"; src = "/nix/store/qq7f6b2wr48qqp9igp5amhk25k11sng9-nixos-wsl-recovery/bin/nixos-wsl-recovery"; } { copy = false; name = "uname"; src = "/nix/store/vwkvhj69z4qqgmpa2lwm97kabf12p26r-coreutils-9.3/bin/uname"; } ]
```